src/utils/questionnaireBuilderUtils.ts

```diff
@@ -26,38 +26,34 @@ export const handleQuestionnaireInputChange = (
  * 
  * @param tagInput - The tag to be added
  * @param questionnaire - The current questionnaire state object
+ * @param strings - The localization strings object
  * @returns Object containing updated questionnaire and any error message
  */
 export const addTag = (
   tagInput: string,
-  questionnaire: Questionnaire
+  questionnaire: Questionnaire,
+  strings: any
```

Comment:
You should not need to use any here. Are you sure you even need to pass it as an argument? IN some cases we do in which case you should be able to find examples of an appropriate type.